backend/src/main/java/ai/verta/modeldb/experiment/ExperimentDAORdbImpl.java

```diff
@@ -584,7 +584,7 @@ public Experiment getExperiment(List<KeyValue> keyValues) throws InvalidProtocol
           stringQueryBuilder.append(" AND ");
         }
       }
-      Query query = session.createQuery(stringQueryBuilder.toString());
+      Query query = session.createQuery(stringQueryBuilder.toString() + " AND ee.deleted = false ");
```

Comment:
Should we use ModelDBConstants.DELETED here too?

Reply:
updated with constant name